basics/migration.md

```diff
@@ -49,6 +49,12 @@ Rari Capital will only accept V2 tokens once they are deployed, so you would
 need to migrate if you want to spend more tokens (e.g. make new deposits) on these
 platforms.
 
+{% hint style="info" %}
+Again, you would not miss any rebase rewards if you don't migrate during the migration
+phase. Even if you chose to migrate way later, you will receive the same amount
```

Comment:
I meant this sentence "Even if you chose to migrate way later, you will receive the same amount
of gOHM that you would have received had you migrated immediately". Does this mean if you migrate later, you will not receive any rebases when migration? I don't think it's the case

Reply:
@Xosten no, the sentence is trying to tell that there is no difference between migrate later vs. migrate immediately

Reply:
@Xosten wdyt? if it is OK can you change this to approved?

Reply:
The conversation is resolved as the sentence is outdated